resources/ajax_htmldata/getIssuesList.php

```diff
@@ -80,7 +80,7 @@ function generateIssueHTML($issue,$associatedEntities=null) {
 
 			if(count($orgIssues) > 0) {
 				foreach ($orgIssues as $issue) {
-					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->organizationID,"entityType"=>1)));
+					echo generateIssueHTML($issue,array(array("name"=>$orgData['organization'],"id"=>$organization->primaryKey,"entityType"=>1)));
```

Comment:
That block of code was originally copied over from the Organizations module's version of the same Issues functionality, where the organizationID property returns the Org's ID.

The resources' version of the Organization class doesn't seem to do the same. I assumed it was just a module difference.

I'll look into it further.